src/dmd/backend/dwarfdbginf.d

```diff
@@ -1815,7 +1815,9 @@ static if (1)
         // See if there are any parameters
         int haveparameters = 0;
         uint formalcode = 0;
-        uint autocode = 0;
+        uint variablecode = 0;
```

Comment:
I renamed this, as I don't see any logic on `autocode` name. If someone have any rationale to maintain that name, please tell. This stores the abbreviation code of formal variables, so `variablecode` seems more sensible to me.